db/databaseconnection.js

```diff
@@ -0,0 +1,29 @@
+const { Client } = require('pg')
```

Comment:
would be interested to know why the Client is in { } as this is not something I am familiar with